pkgs/development/libraries/libgdata/default.nix

```diff
@@ -55,6 +55,7 @@ stdenv.mkDerivation rec {
   ];
 
   propagatedBuildInputs = [
+    gnome3.gnome-online-accounts
```

Comment:
Probably should remove this from `buildInputs` then.

Reply:
Good point, updated.